SpaceShooterGame/SpaceShooterGame/GameScene.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  GameScene.swift
+//  SpaceShooterGame
+//
+//  Created by Tristan Garzon on 2018-03-07.
+//  Copyright © 2018 Tristan Garzon. All rights reserved.
+//
+
+
+//Notes: Timer, Enemies, Firing
+
+import SpriteKit
+import GameplayKit
+
+class GameScene: SKScene, SKPhysicsContactDelegate {
+    
+    
+    var gbackground:SKEmitterNode!
+    var player:SKSpriteNode!
+    
+    //Score
+    var scoreLabel:SKLabelNode!
```

Comment:
Using the ! at the end, makes it so you don't need to write an init, but it also means that if you call it before you set it you'll have a null pointer error. So be careful.